frontend/src/components/baseComponents/ContributeForm.js

```diff
@@ -6,7 +6,8 @@ import Proposal from "../../abis/Proposal.json";
 import { useNavigate } from "react-router-dom";
 import { ethDenverTheme } from "../../ethDenverTheme";
 
-const ContributeButton = () => {
+const ContributeButton = ({ amountRemaining }) => {
+  console.log(amountRemaining);
```

Comment:
I am terrible at this too. Now that its deployed can we try to check each other on consoles?